server/src/services/queue/sqs.js

```diff
@@ -13,12 +13,12 @@ const makeMessageGroupId = msg => {
 
 const generateDeduplicationId = () => crypto.randomBytes(64).toString('hex')
 
-const sendMessage = async (msg, options = {}) => {
+const sendMessage = async (msg) => {
   const params = {
     MessageBody: JSON.stringify(msg),
     QueueUrl: queueUrl,
     MessageGroupId: makeMessageGroupId(msg),
-    ...(options.generateDeduplicationId && { MessageDeduplicationId: generateDeduplicationId() })
+    MessageDeduplicationId: get(msg, 'params.correlationId') || generateDeduplicationId()
```

Comment:
@leonprou the `get` function gets third argument which will be the default if the second value wasn't found, so you can pass `generateDeduplicationId()` as default